src/JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -306,6 +306,12 @@ private function denormalizeRelation(string $resourceClass, string $attributeNam
             return $this->serializer->denormalize($value, $className, self::FORMAT, $this->createRelationContext($className, $context));
         }
 
+        if (!is_object($value) && !is_array($value)) {
```

Comment:
Actually, I'm not sure anymore after symfony/symfony#17660

@dunglas Are we due for a rewrite?